dsgrid/units/energy.py

```diff
@@ -0,0 +1,178 @@
+"""Contains functions to convert electricity between units."""
+
+import logging
+
+import pyspark.sql.functions as F
+from pyspark.sql import DataFrame
+
+from dsgrid.exceptions import DSGInvalidDimension, DSGInvalidParameter
+
+
+KWH = "kWh"
+MWH = "MWh"
+GWH = "GWh"
+TWH = "TWh"
+THERM = "therm"
+MBTU = "MBtu"
+KWH_PER_THERM = 29.3001111
+KWH_PER_MBTU = 0.293071
+THERMS_PER_MBTU = 0.01
```

Comment:
@lixiliu, what version of "therm" should we be using? And what reference should we (maybe) provide here in a comment? Per https://en.wikipedia.org/wiki/Therm, this appears to be the "US" version of Therm with BTU defined by water at 59 F (https://en.wikipedia.org/wiki/British_thermal_unit).

(I also appreciate the latter link's quick note that BTU definitions vary by up to 0.5%.)